### PR TITLE
Add a mapping helper for welding airlocks

### PR DESCRIPTION
**Pull request by Maurukas**
## About The Pull Request
What it says on the tin. Warning: copy-pasted mapper sprites.

## Why It's Good For The Game

Fewer varedits, better maintainability, more obvious in the map editor what's been applied to a given airlock.

## Changelog

No player facing changes.

### CODE REVIEW

**Comment by Jolly-66** [comment]
Bless being able to weld airlocks instead of using wooden barricades 
Good PR

File: code/modules/mapping/mapping_helpers.dm [hunk 1]
```diff
@@ -216,6 +216,16 @@
...
 
+/obj/effect/mapping_helpers/airlock/welded
+	name = "airlock welded helper"
+	icon_state = "airlock_welded"
+
+/obj/effect/mapping_helpers/airlock/welded/payload(obj/machinery/door/airlock/airlock)
+	if(airlock.welded)
+		log_mapping("[src] at [AREACOORD(src)] tried to make [airlock] welded but it's already welded closed!")
+	else
+		airlock.welded = TRUE
```
Comment (san7890) [inline code/modules/mapping/mapping_helpers.dm:227]: does it show up as welded in-game? might have to do `update_appearance()` on the `airlock`

Comment (Maurukas) [reply code/modules/mapping/mapping_helpers.dm:227]: It's in LateInitialize() on the parent. Also I tested it.

File: code/modules/mapping/mapping_helpers.dm [hunk 2]
```diff
@@ -216,6 +216,16 @@
...
 		airlock.abandoned = TRUE
 
+/obj/effect/mapping_helpers/airlock/welded
+	name = "airlock welded helper"
+	icon_state = "airlock_welded"
+
+/obj/effect/mapping_helpers/airlock/welded/payload(obj/machinery/door/airlock/airlock)
+	if(airlock.welded)
+		log_mapping("[src] at [AREACOORD(src)] tried to make [airlock] welded but it's already welded closed!")
+	else
```
Comment (Mothblocks) [inline code/modules/mapping/mapping_helpers.dm:226]: Should remove the else because it's unnecessary and makes this slightly harder to read

**Comment by san7890** [comment]
image

CI is failing, that's my fault i forgot to mention that the updatepaths i fed you was just an example because i didn't bother to copy the real path

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 3]
```diff
@@ -8768,9 +8768,9 @@
 /area/ruin/space/ancientstation/beta/atmos)
 "XD" = (
 /obj/machinery/door/airlock/maintenance_hatch{
-	name = "Custodial Closet";
-	welded = 1
+	name = "Custodial Closet"
 	},
+/obj/effect/mapping_helper/welded,
```
Comment (san7890) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm:8773]:
```suggestion
/obj/effect/mapping_helpers/airlock/welded,
```

**Pull request merged**